packages/sources/nav-consulting/src/transport/fundNav.ts

```diff
@@ -0,0 +1,50 @@
+import { Requester } from '@chainlink/external-adapter-framework/util/requester'
+import { getValidatorIds } from './authentication'
+import { AdapterError } from '@chainlink/external-adapter-framework/validation/error'
+
+import { format } from 'date-fns'
+import { utc } from '@date-fns/utc'
+
+interface Response {
+  Data: {
+    Date: string
+    'Ending Net Asset Value': {
+      DTD: string
+    }
+  }[]
+}
+
+const getDateString = (raw: string) => format(new Date(raw), 'MM-dd-yyyy', { in: utc })
+
+export const getFundNav = async (
+  globalFundID: string,
+  date: string,
+  url: string,
+  apiKey: string,
+  secret: string,
+  requester: Requester,
+) => {
```

Comment:
maybe worth specifying a return type and returning a number here?

Reply:
Let me see if I can type the raw response into number